perceval/backends/puppet/puppetforge.py

```diff
@@ -205,7 +207,7 @@ def parse_json(raw_json):
     def _init_client(self, from_archive=False):
         """Init client"""
 
-        return PuppetForgeClient(PUPPET_FORGE_URL, self.max_items, self.archive, from_archive)
+        return PuppetForgeClient(PUPPET_FORGE_URL, self.max_items, self.archive, from_archive, ssl_verify=True)
```

Comment:
This should be `ssl_verify=self.ssl_verify`, like in the github backend: https://github.com/chaoss/grimoirelab-perceval/commit/068b1fa91e3bea8dc6e62d734ef4aaa9ba7eecdf#diff-06eac4618b889067c3a258f5c69384bdR286